routers/api/v1/repo/commits.go

```diff
@@ -90,10 +98,6 @@ func GetAllCommits(ctx *context.APIContext) {
 	//   in: query
```

Comment:
https://github.com/go-gitea/gitea/pull/9452#issuecomment-568289818 ... and here you can see the mix i write about ...